src/style.scss

```diff
@@ -18,12 +18,31 @@
   color: currentColor;
   vertical-align: middle;
   line-height: 1;
+  display: inline-block;
 
   .logo {
     color: var(--ds-color-brand-midnight-400, $ds-color-brand-midnight-400);
   }
 }
 
+:host([customSize]) {
+  --auro-size-lg:100%
```

Comment:
@jason-capsule42 Hey, it looks like the missing semi-colon got copied over to the new file. So https://github.com/AlaskaAirlines/auro-icon/issues/128 is still a problem